docker-compose.build.yml

```diff
@@ -0,0 +1,93 @@
+services:
+  wikibase:
+    image: ${WIKIBASE_SUITE_WIKIBASE_IMAGE_URL?}:${WBS_WIKIBASE_VERSION?}
```

Comment:
The `?` tell yaml/compose to throw an error if the value doesn't exist. They are used throughout this file to avoid any accidental building of images which don't for some reason have all the env vars they need.